e2e/cypress/tests/support/notification.ts

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+export interface NotificationWindow extends Window {
+    Notification(title: string, opts?: NotificationOptions): void;
+}
```

Comment:
0/5 on exporting this. since this might only be used for this particular use case.